CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
-# dbt_netsuite_source version.version
+# dbt_netsuite_source v0.12.0
+
+## Breaking Schema Changes
+- Added optional `fiscalcalendar` source table to support accurate fiscal year start dates (currently defaulted to calendar year). This table and related models (`stg_netsuite2__fiscal_calendar_tmp` and `stg_netsuite2__fiscal_calendar`) are disabled by default. To enable this feature:
```

Comment:
```suggestion
- Added optional `fiscalcalendar` source table to support accurate fiscal year start dates (currently defaulted to calendar year). This table and related models (`stg_netsuite2__fiscal_calendar_tmp` and `stg_netsuite2__fiscal_calendar`) are disabled by default. ([#61](https://github.com/fivetran/dbt_netsuite_source/pull/61))
- To enable this feature:
```

Reply:
"PR" added, but I prefer the structure where each individual update is detailing a change and how to take advantage of it. Keeping the "To enable this feature:" within the same primary bullet as before and letting the sub-bullets be the enable steps.